src/Microsoft.DotNet.Build.CloudTestTasks/RunnerScripts/xunitrunner-perf/xunitrunner-perf.py

```diff
@@ -74,9 +74,41 @@ def _prepare_execution_environment(settings, framework_in_tpa, assembly_list_nam
     log.info("Copying product binaries from {} to {}".format(build_drop, framework_target))
     _copy_package_files(assembly_list, build_drop, framework_target, core_root, test_location)
 
+def _copy_files_to_dest(src, dest):
+    for item in os.listdir(src):
+        s = os.path.join(src, item)
+        d = os.path.join(dest, item)
+        if os.path.isfile(s):
+            shutil.copy2(s, d)
+
+def _prepare_linux_env_for_perf(correlation_dir, xunit_perf_drop, test_location):
+    # copy over the cli runner and its dependencies for execution
+    _copy_files_to_dest(xunit_perf_drop, test_location)
+    xunit_perf_deps = os.path.join(correlation_dir,"Microsoft.DotNet.xunit.performance.run.core")
+    xunit_perf_deps = os.path.join(xunit_perf_deps, os.listdir(xunit_perf_deps)[0])
```

Comment:
I'd probably have some sort of log or warning to make sure this directory only has a single file in it.